### DISCUSSION TITLE
Relation "orders not showing up in api response

### DISCUSSION THREAD

**Discussion by Ermal45** [strapi/strapi#7867] [Legacy Topics]
Hello everyone. So i have a problem that i dont know how to get around it.I have a content type which is called orders, so every time a order gets created, i use a id which is coming form user context and it tells which user has relation with orders.Now i want to fetch orders in front end with axios by using users.me but i don't see orders anywhere in  my response.I would appreciate if anyone could help me thanks.
![Screenshot (86)](https://user-images.githubusercontent.com/62566277/93001795-e147c080-f531-11ea-8a8a-e4fca8870b17.png)

**Comment by derrickmehaffy** [comment]
Hello!

By default the `/users/me` has a population only set for `roles` you will need to use the [extensions ](https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#how-do-i-customize-a-plugin)to modify the following file and adjust the population to fit your needs:

https://github.com/strapi/strapi/blob/cd9296b37bca030aa1c56e4782a28f1ce79a24f8/packages/strapi-plugin-users-permissions/services/User.js#L68-L74

**Reply by derrickmehaffy** [reply]
In the above example you can see we are only sending the population array as `['role']` you will need to add in your relational fields you wish to populate.
